src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String[]> result = readDataFromFile(fromFileName);
+        int[] statistics = calculate(result);
+        writeDataToFile(toFileName, statistics);
+
+    }
+
+    private List<String[]> readDataFromFile(String fromFileName) {
+        List<String[]> result = new ArrayList<>();
+        try {
+            List<String> lines = Files.readAllLines(Paths.get(fromFileName));
+            for (String line : lines) {
+                String[] values = line.split(",");
+                result.add(values);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file " + fromFileName, e);
+        }
+        return result;
+    }
```

Comment:
Do not mix **reading** with **data processing**
```suggestion
    private List<String[]> readDataFromFile(String fromFileName) {
        try {
            return Files.readAllLines(Paths.get(fromFileName));
        } catch (IOException e) {
            throw new RuntimeException("Can't read data from the file " + fromFileName, e);
        }
    }
```